docs/EN_US/ECLLanguageReference/ECLR_mods/BltInFunc-BUILD.xml

```diff
@@ -807,6 +808,89 @@ END;
 #WORKUNIT('name','Ppass.FilterDsLib')
 BUILD(FilterDsLib1);
 </programlisting>
+  </sect2>
+
+  <sect2 id="IDXCOMP2" role="brk">
+    <title>Indexes and Compression</title>
+
+    <para><emphasis role="bold">Compression Options:</emphasis></para>
+
+    <informaltable colsep="1" frame="all" rowsep="1">
+      <tgroup cols="2">
+        <colspec align="left" colwidth="122.40pt" />
+
+        <colspec />
+
+        <tbody>
+          <row>
+            <entry><emphasis role="bold">LZW</emphasis></entry>
+
+            <entry>The default compression. It is a variant of the
+            Lempel-Ziv-Welch algorithm. This is the original index compression
+            format (which is less efficient than the inplace compression
+            format). It remains the default for backward
+            compatibility.</entry>
+          </row>
+
+          <row>
+            <entry><emphasis role="bold">ROW</emphasis></entry>
+
+            <entry>Compresses index entries based on differences between
+            contiguous rows (<emphasis role="bold">for use with fixed-length
```

Comment:
contiguous? clearer if that word is deleted.  Also in following paragraph I'm not sure we would recommend it..

 Compresses index entries based on differences between
            rows (<emphasis role="bold">for use with fixed-length
            records only</emphasis>). It typically does not compress
as well as LZW, but takes up less space in memory because the rows are expanded on demand.